### PR TITLE
Updated Readme for Effective Reading

### PR DESCRIPTION
**Pull request by Piyussshh**
Changed 
1. "Term"  to "Terms"  
2. "object oriented"  to "object-oriented"
3. "less" to "fewer"

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -16,14 +16,14 @@
 ## Key features
 
 ArkScript is
-* small: the compiler, and the virtual machines fit under 5000 lines, but also small in term of keywords (it has only 10)!
+* small: the compiler and the virtual machines fit under 5000 lines, but also small in terms of keywords (it has only 10)!
```
Comment (SuperFola) [inline README.md:19]: Just a small comment but you couldn't know (and I'm taking the opportunity to hit two birds in one stone): it's 8000 lines now, not 5000

**Review by SuperFola — CHANGES_REQUESTED** [review]

**Comment by Piyussshh** [comment]
Yes?

**Comment by Piyussshh** [comment]
> @Piyussshh ?

Yes? @SuperFola

**Comment by Piyussshh** [comment]
> Could you correct your PR according to the comments I left?

on it!

**Review by SuperFola — APPROVED** [review]

**Pull request merged**